main.py

```diff
@@ -352,6 +354,29 @@ def missing_page(campaign_group=None):
                            user_info=user_info,
                            short_names=short_names)
 
+@app.route('/missing_update', methods=['POST'])
+def missing_update():
+    """
+    Endpoint to update interested notes for missing samples of the twiki database
+    """
+    data = json.loads(request.data)
+    update_time = int(time.time())
+    logging.info(data)
+    #Updating notes for missing samples
+    missing_dataset_name = data['dataset_name']
+    missing_campaign = data['campaign']
+    twiki_conn = sqlite3.connect('twiki.db')
+    twiki_cursor = twiki_conn.cursor()
+    if 'missing_nts' in data:
```

Comment:
I would suggest to remove this `if` from here and add an opposite one, just after `data = json.loads(...)`.
```
if 'missing_nts' not in data:
    return 'Request does not have notes', 400  # Of course, you should word error message better...

# Do sqlite connect, cursor, commits, etc.
```
This will do all these complex things like creating a connection to DB, getting a cursor, doing a commit, etc. only if there are actual notes to be updated. There is no point in opening a connection to a database just to find out that we don't actually have notes to update? :alien: